src/displayapp/LittleVgl.cpp

```diff
@@ -12,6 +14,45 @@ namespace {
     lv_theme_t* theme = lv_pinetime_theme_init();
     lv_theme_set_act(theme);
   }
+
+  namespace {
```

Comment:
Anonymous namespace twice is a bit strange.

Reply:
Oh yes, right... let me fix this...